Define the handling of invalid sequence numbers found in QX_PING frames (#52)
kazuho/draft-kazuho-quic-quic-services-for-streams: draft-ietf-quic-qmux.md

type 0x348c67529ef8c7be carrying the largest Sequence Number that the endpoint
has received.

If an endpoint receives a QX_PING frame of type 0x348c67529ef8c7be with a
sequence number greater than the greatest sequence number that it sent in a
QX_PING frame of type 0x348c67529ef8c7bd, the endpoint MUST close the connection
with an error of type PROTOCOL_VIOLATION. Similarly, an endpoint MAY detect a
sequence number that it did not send and close the connection with the same
error.

If an endpoint receives a QX_PING frame of type 0x348c67529ef8c7bd with a

> sequence number no greater than the previous sequence number received in a frame

Reviewer: A small nit-pick, this reads a bit better to me.
```suggestion
sequence number not greater than the previous sequence number received in a frame
```

Reviewer: Alternatively:
```suggestion
sequence number less than or equal to the previous sequence number received in a frame
```

(apologies if this is too nit-picky)
